### PR TITLE
[13.0][IMP] hr_attendance_report_theoretical_time: Separate the setup() of the tests in TestHrAttendanceReportTheoreticalTimeBase to improve the inheritance of tests in other modules.

### PR DESCRIPTION
**Pull request by victoralmau**
Separate the `setup()` of the tests in `TestHrAttendanceReportTheoreticalTimeBase` to improve the inheritance of tests in other modules.

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT34898

### CODE REVIEW

**Comment by pedrobaeza** [comment]
For avoiding this high diff, put both classes in the same file.

**Review by flotho — APPROVED** [review]
make sense, code review

File: hr_attendance_report_theoretical_time/tests/test_hr_attendance_report_theoretical_time.py [hunk 1]
```diff
@@ -136,6 +136,12 @@ def setUpClass(cls):
                     )
                 )
 
+
+class TestHrAttendanceReportTheoreticalTime(TestHrAttendanceReportTheoreticalTimeBase):
+    @classmethod
```
Comment (pedrobaeza) [inline hr_attendance_report_theoretical_time/tests/test_hr_attendance_report_theoretical_time.py]: No need to put this empty method

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**